dod-ccpo/atat-web-ui
# PR 1924 - feature/AT-9641-feedback-widget-for-non-ditco-user

Adds feedback form to review documents page for non-DITCO users

Instructions to test:
- On the first page of a new acquisition package, select "No. I plan to use another Contracting Office."
- Navigate to Step 9 - Generate Package Documents and click the "Generate my acquisition package" button
- Go to your SNOW instance, find open your acquisition package in the list of `DAPPS:Acquisition Packages`, move DOCGEN Job Status from In Progress to Success, and click Update to save the package
- Back in your local app, you'll now be on the page to observe the feedback widget for non-DITCO users

![image](https://github.com/dod-ccpo/atat-web-ui/assets/135259059/43c7e174-eaba-4a4e-9381-bbcdc3be9aad)
